packages/contextual-menu/contextual-menu.scss

```diff
@@ -0,0 +1,92 @@
+@use "~@fremtind/jkl-core/jkl";
+@use "contextual-menu-item";
+@use "contextual-menu-trigger-icon";
+
+@include jkl.light-mode-variables {
+    --jkl-contextual-menu--border-color: #{jkl.$color-skifer};
+    --jkl-contextual-menu-bg-color: #{jkl.$color-hvit};
+    --jkl-contextual-menu-item-hover-background-color: #{jkl.$color-sand};
+    --jkl-contextual-menu-item-trigger-font-color: #{jkl.$color-svart};
+}
+
+@include jkl.dark-mode-variables {
+    --jkl-contextual-menu--border-color: #{jkl.$color-hvit};
+    --jkl-contextual-menu-bg-color: #{jkl.$color-svart};
+    --jkl-contextual-menu-item-hover-background-color: #{jkl.$color-skifer};
+    --jkl-contextual-menu-item-border: 2px solid #{jkl.$color-snohvit};
+    --jkl-contextual-menu-item-trigger-font-color: #{jkl.$color-hvit};
+}
+
+@include jkl.comfortable-density-variables {
+    --jkl-contextual-menu-item-font-size: jkl.$typography-font-size-3;
+    --jkl-contextual-menu-item-font-weight: jkl.$typography-weight-normal;
+    --jkl-contextual-menu-menu-item-height: jkl.rem(48px);
+}
+
+@include jkl.compact-density-variables {
+    --jkl-contextual-menu-item-font-size: jkl.$typography-font-size-1;
+    --jkl-contextual-menu-item-font-weight: jkl.$typography-weight-normal;
+    --jkl-contextual-menu-menu-item-height: jkl.rem(32px);
+}
+
+.jkl-contextual-menu {
+    width: fit-content;
+    display: flex;
+    align-items: center;
+    max-width: jkl.rem(300px);
+    margin: 0;
+
+    &__trigger-btn {
+        @include jkl.reset-outline;
```

Comment:
Med `reset-outline` må vi sørge for å gi en custom focus-styling. Sånn det er nå mister jeg oversikten over hvor fokuset er når jeg `Tab`er til triggerknappen.

Lurer på om vi kanskje bare fjerner denne, og lar fokus-stylingen være opp til trigger-elementet?

```suggestion
```